packages/common/src/models/User.ts

```diff
@@ -58,6 +58,7 @@ export type UserMetadata = {
   tikTokVerified?: boolean
   balance?: Nullable<StringWei>
   total_balance?: Nullable<StringWei>
+  usdc_balance?: Nullable<StringWei>
```

Comment:
I think `StringUSDC`? Alternatively this could live in it's own slice (since only the current user will have this field populated), but no strong conviction either way there

Reply:
Yeah I think this only needs to live here if we're planning on returning it with the User response from DN. If we're fetching the balance from chain directly, this can go.
I'll remove it for now until we implement that feature.